operator/v1/types_network.go

```diff
@@ -507,6 +507,65 @@ type GatewayConfig struct {
 	// The supported values are "Restricted" and "Global".
 	// +optional
 	IPForwarding IPForwardingMode `json:"ipForwarding,omitempty"`
+	// ipv4 allows users to configure IP settings for IPv4 connections. When omitted, this means no opinion and the defualt
+	// configuration is used. Check individual members fields within ipv4 for details of default values.
+	// +optional
+	IPv4 *IPv4GatewayConfig `json:"ipv4,omitempty"`
```

Comment:
Shouldn't be a pointer unless it contains required fields, also doesn't need omitempty
```suggestion
	IPv4 IPv4GatewayConfig `json:"ipv4"`
```